### ISSUE TITLE
Accessibility: reordering menu items should have some indication about the item level

### ISSUE THREAD

**Issue by afercia** [voldemortensen/menu-customizer#105]
Splitting this out from the users test post on Make WordPress Accessible:
https://make.wordpress.org/accessibility/2015/06/12/accessibility-usertest-menu-customizer/

When reordering menu items, there should be an audible indication of their level.

NVDA
When reordering menu items, NVDA anounces “menu is now a submenu item” or “menu item moved out of submenu”, which is useful, However, when reviewing the list while reordering, there is no indication at what level any given menu item is currently set to.

Supernova
When using the reorder feature, there is no screen reader feedback to indicate the menus level, perhaps a level indicator can be added after the page title that says something like “top”, “level 1″, “level 2″, etc to show how far down the tree each menu item is. As this is only really to benefit screen reader users, it could be visually hidden using the CSS clip method.

**Comment by designsimply** [comment]
Moved to https://core.trac.wordpress.org/ticket/32728
